sof_wrapper/api/fhir.py

```diff
@@ -108,10 +116,18 @@ def pdmp_med_orders(**kwargs):
         base_url=current_app.config['PDMP_URL'],
         )
     params = kwargs or request.args
+    # decoded JWT, or FHIR Practioner reference (eg Practitioner/ID)
     user = get_session_value('user')
-    if not user or 'DEA' not in user:
+
+    if user and "DEA" in user:
+        DEA = user["DEA"]
+    # in a demo deploy, SCRIPT_ENDPOINT_URL will be configured, but empty
+    elif current_app.config.get("SCRIPT_ENDPOINT_URL") == "":
+        DEA = "FAKEDEA123"
+    else:
         return jsonify_abort(status_code=400, message="DEA not found")
-    params['DEA'] = user['DEA']
+
+    params['DEA'] = DEA
```

Comment:
As this has grown to be more than a few lines, it'd be nice to DRY it out a bit with a local function - say:
```
def get_dea(user):
 [...]

params['DEA'] = get_dea(user)

Reply:
@pbugni fixed in 212f767a00630354d66b5f0ce39b5fb75141939c